src/CMakeLists.txt

```diff
@@ -15,6 +15,10 @@ ADD_SUBDIRECTORY(dreln)
 ADD_SUBDIRECTORY(drm)
 ADD_SUBDIRECTORY(dstripe)
 ADD_SUBDIRECTORY(dsync)
+IF(ENABLE_HDF5)
+  ADD_SUBDIRECTORY(daos-serialize)
+  ADD_SUBDIRECTORY(daos-deserialize)
+ENDIF(ENABLE_HDF5)
```

Comment:
@dsikich Maybe we should also make sure ENABLE_DAOS is turned on before adding these tools, just in case a user tries to compile with HDF5 support but not DAOS support.